webapp/src/pages/FoodCheckoutConfirmation.jsx

```diff
@@ -49,53 +50,77 @@ export default function FoodCheckoutConfirmation() {
         <p className="mb-0">{t("food:confirmation_subtitle")}</p>
       </div>
       <LayoutContainer gutters top>
-        <h4 className="mb-3">{t("food:checkout_items_title")}</h4>
+        <h4 className="mb-3">{t("food:confirmation_my_order")}</h4>
         {items.map((p, idx) => (
           <Item key={idx} item={p} />
         ))}
+        {user.unclaimedOrdersCount !== 0 && (
+          <FormButtons
+            className="mb-4"
+            primaryProps={{
+              type: "button",
+              variant: "success",
+              children: t("food:unclaimed_order_history_title"),
+              to: "/unclaimed-orders",
+              as: RLink,
+            }}
+          />
+        )}
+        {offering.fulfillmentInstructions && (
+          <p className="lead">{offering.fulfillmentInstructions}</p>
+        )}
       </LayoutContainer>
-      <hr />
-      <LayoutContainer gutters>
-        <h4>{offering.fulfillmentConfirmation}</h4>
-        <p>{fulfillmentOption.description}</p>
-      </LayoutContainer>
-      <hr />
+      <hr className="my-4" />
+      {fulfillmentOption && (
+        <>
+          <LayoutContainer gutters>
+            <h4>{offering.fulfillmentConfirmation}</h4>
+            <p>{fulfillmentOption.description}</p>
+          </LayoutContainer>
+          <hr className="my-4" />
+        </>
+      )}
       <LayoutContainer gutters>
-        <p>{t("food:confirmation_message")}</p>
+        <h4>{t("food:confirmation_transportation_title")}</h4>
+        <p className="mb-0">{t("food:confirmation_transportation_subtitle")}</p>
         <FormButtons
```

Comment:
As above, this is not really a form either. There's no form, inputs, or secondary action that would be a reason to use this component. It's just a single centered button, AFAICT.